project.clj

```diff
@@ -7,11 +7,12 @@
   :dependencies [[org.clojure/clojure "1.9.0-alpha17"]
                  [org.clojure/data.xml "0.0.8"] ; parse xml lazily
                  [org.clojure/data.int-map "0.2.4"] ; fast integers-map
-                 [metosin/compojure-api "2.0.0-alpha5"]
-                 [metosin/spec-tools "0.3.0"]
+                 [metosin/compojure-api "2.0.0-alpha7"]
+                 [metosin/spec-tools "0.3.2"]
                  [org.clojure/test.check "0.9.0"]]
   :profiles {:dev {:dependencies [[criterium "0.4.4"]  ;; benchmark
-                                  [expound "0.1.1"]]}}
+                                  [expound "0.1.1"]
+                                  [io.aviso/pretty "0.1.34"]]}}
```

Comment:
@mehdisadeghi I introduced this because the stack traces in travis-ci were impossible to decipher. Cursive does a great job at reducing the noise from stacktraces but well travis ci is left alone for that.

Aviso pretty is an alternative way of displaying stacktraces. Not sure if we want that in our everyday development but I think it would be interesting to give it a try.

Reply:
I think it is harmless. We can remove it any time we like just by deleting a few line of code from the project file. So let's leave it there.